library/neo4j.py

```diff
@@ -132,12 +132,22 @@ def process_org_chart(self, org: list[Employee]):
     def get_chief_executives(self) -> list[Employee]:
         query = """
         MATCH (w:Person)
-        WHERE NOT EXISTS( (w)-[:REPORTS_TO]->() )
+        WHERE w.employee_id IS NOT NULL AND NOT EXISTS( (w)-[:REPORTS_TO]->() )
         RETURN w as ceo"""
         with self.driver.session() as session:
-            results: list[Record] = list(session.run(query))        
+            results: list[Record] = list(session.run(query))     
+            [print(r) for r in results]   
             return [Employee(**record['ceo']) for record in results]
         
+    def get_employee_with_full_org_chart(self, email: str) -> Employee:
+        result = self.get_org_chart_below(email)
+        if result is None:
+            return None
+        managers = self.get_org_chart_above(email)
+        if len(managers) > 1:
+            result.manager = managers[1]
```

Comment:
I noticed that you add the employee in the result when this function get_org_chart_above is called. Is that for a reason?

Reply:
The org chart above is a linked list, the org chart below is a tree. In both functions, the first entry is you.

So if you are the CEO, your list above has a single entry, yourself. 

There should be some clarity on this structure in the tests.

Reply:
Okay got it! Thank you.